torch_xla/csrc/tensor.cpp

```diff
@@ -1671,4 +1683,17 @@ xla::uint64 XLATensor::GetRunningSeed(const Device& device) {
   return DeviceContextArena::Get()->GetRunningSeed(device);
 }
 
+bool XLATensor::UseEagerDebugMode() {
```

Comment:
hmm.. this doesn't needs to be a class method. I don't want to block you because of this but maybe I will fix it later.